docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,64 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: This page compares a function execution in a traditional system and a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
```

Comment:
OK, so it turns out we don't need this at all :)

Reply:
I'll yank it in my next commit.